pom.xml

```diff
@@ -1,859 +1,86 @@
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
-  <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.jboss</groupId>
-  <version>30-SNAPSHOT</version>
-  <artifactId>jboss-parent</artifactId>
+    <groupId>org.jboss</groupId>
+    <version>30-SNAPSHOT</version>
+    <artifactId>jboss-parent-parent</artifactId>
 
-  <packaging>pom</packaging>
+    <packaging>pom</packaging>
 
-  <name>JBoss Parent POM</name>
-  <description>Parent POM for JBoss projects. Provides default project build configuration.</description>
-  <url>http://www.jboss.org</url>
-
-  <issueManagement>
-    <system>JIRA</system>
-    <url>https://issues.jboss.org/</url>
-  </issueManagement>
-
-  <scm>
-    <connection>scm:git:git@github.com:jboss/jboss-parent-pom.git</connection>
-    <developerConnection>scm:git:git@github.com:jboss/jboss-parent-pom.git</developerConnection>
-    <url>http://github.com/jboss/jboss-parent-pom</url>
-    <tag>HEAD</tag>
-  </scm>
-
-  <developers>
-    <developer>
-      <id>jboss.org</id>
-      <name>JBoss.org Community</name>
-      <organization>JBoss.org</organization>
-      <organizationUrl>http://www.jboss.org</organizationUrl>
-    </developer>
-  </developers>
-
-  <mailingLists>
-    <mailingList>
-      <name>JBoss User List</name>
-      <subscribe>https://lists.jboss.org/mailman/listinfo/jboss-user</subscribe>
-      <unsubscribe>https://lists.jboss.org/mailman/listinfo/jboss-user</unsubscribe>
-      <archive>http://lists.jboss.org/pipermail/jboss-user/</archive>
-    </mailingList>
-    <mailingList>
-      <name>JBoss Developer List</name>
-      <subscribe>https://lists.jboss.org/mailman/listinfo/jboss-development</subscribe>
-      <unsubscribe>https://lists.jboss.org/mailman/listinfo/jboss-development</unsubscribe>
-      <archive>http://lists.jboss.org/pipermail/jboss-development/</archive>
-    </mailingList>
-  </mailingLists>
-
-  <licenses>
-    <license>
-      <name>Public Domain</name>
-      <url>http://repository.jboss.org/licenses/cc0-1.0.txt</url>
-      <distribution>repo</distribution>
-    </license>
-  </licenses>
-
-  <organization>
-    <name>JBoss by Red Hat</name>
+    <name>JBoss Parent Parent POM</name>
+    <description>Parent Parent POM for JBoss projects. Provides default project build configuration.</description>
```

Comment:
Perhaps a bit more words about the structure? For example

"Provides, via child modules, default project build configurations, both for projects producing non-compiled artifacts and traditional ones compiled using a single JDK version, and also for projects producing multi-release jars with resources compiled using multiple JDK versions."

Reply:
Sure, I was pretty lazy on this one :)

Reply:
Fixed.

Reply:
Well, I think my 32 added words is a smaller contribution than yours! ;)